src/libsystemd-network/sd-dhcp-server.c

```diff
@@ -282,9 +291,10 @@ static int dhcp_server_send_udp(sd_dhcp_server *server, be32_t destination,
                 .msg_namelen = sizeof(dest.in),
                 .msg_iov = &iov,
                 .msg_iovlen = 1,
-                .msg_control = &control,
-                .msg_controllen = sizeof(control),
+                .msg_control = NULL,
+                .msg_controllen = 0,
```

Comment:
structured initialization in C defaults to zero initialize unlisted fields, hence just drop these two lines